shared/src/commonMain/kotlin/com/karumi/gallery/domain/TimeProvider.kt

```diff
@@ -0,0 +1,8 @@
+package com.karumi.gallery.domain
+
+expect class TimeProvider {
+
+  fun persistTime()
+  fun getPersistedTime(): Long
```

Comment:
When talking about time I'd either use stronger typing like `Duration` or specify the unit in the name: `getPersistedTimeInMs`

Reply:
I thought I couldn't use `Duration` in kotlin native 🤔, but I can use it like `System.currentTimeMillis().toDuration(DurationUnit.MILLISECONDS)`

Reply:
Finally, I changed it with `getPersistedTimeInMs`